engine/src/gfx/nav/navscreen.cpp

```diff
@@ -340,15 +340,7 @@ void NavigationSystem::Setup()
         system_item_scale *= (screenskipby4[3]-screenskipby4[2]);
     screenoccupation = new navscreenoccupied( screenskipby4[0], screenskipby4[1], screenskipby4[2], screenskipby4[3], 1 );
 
-    //Get special colors from the config
-    static GFXColor currentcol = vs_config->getColor( "nav", "current_system",
-                                           GFXColor( 1, 0.3, 0.3, 1.0 ) );
-    static GFXColor destinationcol = vs_config->getColor( "nav", "destination_system",
-                                           GFXColor( 1, 0.77, 0.3, 1.0 ) );
-    static GFXColor selectcol = vs_config->getColor( "nav", "selection_system",
-                                           GFXColor( 0.3, 1, 0.3, 1.0 ) );
-    static GFXColor pathcol = vs_config->getColor( "nav", "path_system",
-                                           GFXColor( 1, 0.3, 0.3, 1.0 ) );
```

Comment:
these should probably be here, but not have the `static GFXColor` portion; that basically creates a new variable that then gets dropped when it leaves scope.

Reply:
@evertvorster the above issue might be why you were having issues with the tests you did - not sure, just a guess

Reply:
Let me test your theory.... if it works, then I know where to put them in the config file as well.